### ISSUE TITLE
"Unable to load dynamic library"

### ISSUE THREAD

**Issue by Kamprath** [phalcon/box#37]
After provisioning the image and SSH-ing into the box, running the `phalcon` command fails with message:

> PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php/20131226/phalcon.so' - /usr/lib/php/20131226/phalcon.so: cannot open shared object file: No such file or directory in Unknown on line 0
ERROR: Phalcon extension isn't installed, follow these instructions to install it: https://docs.phalconphp.com/en/latest/reference/install.html

This is on Mac OS 10.11.6, VirtualBox 5.1.18 r114002 (Qt5.6.2), and Vagrant 1.9.3 with the vagrant-vbguest plugin installed.

**Comment by sergeyklay** [comment]
I'll take a look asap.

**Comment by sergeyklay** [comment]
Will be fixed in the https://github.com/phalcon/box/pull/38

**Comment by nurazije** [comment]
A dumb question:
Any information about the date when this will be released?
I tried to merge 2.0.x on my machine with master and it has a lot of other problems.

**Comment by sergeyklay** [comment]
@nurazije I'll try to release today v2.0.0

**Comment by sergeyklay** [comment]
@Kamprath, @nurazije 

I've merged #38 without release a ned tag.
It is still beta, but fully worked.


Could you please try it now?:
```bash
$ vagrant box add phalconphp/xenial64

$ git clone https://github.com/phalcon/box.git $HOME/workspace

$ cd $HOME/workspace

$ bash install

$ vagrant up
```

We are working on documentation and small improvements right now. Feel free to ask any questions if you have them. And please look at README.md
